test/fail_compilation/diag5385.d

```diff
@@ -1,22 +1,14 @@
 /*
 TEST_OUTPUT:
 ---
-fail_compilation/diag5385.d(27): Deprecation: `imports.fail5385.C.privX` is not visible from module `diag5385`
-fail_compilation/diag5385.d(27): Error: class `imports.fail5385.C` member `privX` is not accessible
-fail_compilation/diag5385.d(28): Deprecation: `imports.fail5385.C.packX` is not visible from module `diag5385`
-fail_compilation/diag5385.d(28): Error: class `imports.fail5385.C` member `packX` is not accessible
-fail_compilation/diag5385.d(29): Deprecation: `imports.fail5385.C.privX2` is not visible from module `diag5385`
-fail_compilation/diag5385.d(29): Error: class `imports.fail5385.C` member `privX2` is not accessible
-fail_compilation/diag5385.d(30): Deprecation: `imports.fail5385.C.packX2` is not visible from module `diag5385`
-fail_compilation/diag5385.d(30): Error: class `imports.fail5385.C` member `packX2` is not accessible
-fail_compilation/diag5385.d(31): Deprecation: `imports.fail5385.S.privX` is not visible from module `diag5385`
-fail_compilation/diag5385.d(31): Error: struct `imports.fail5385.S` member `privX` is not accessible
-fail_compilation/diag5385.d(32): Deprecation: `imports.fail5385.S.packX` is not visible from module `diag5385`
-fail_compilation/diag5385.d(32): Error: struct `imports.fail5385.S` member `packX` is not accessible
-fail_compilation/diag5385.d(33): Deprecation: `imports.fail5385.S.privX2` is not visible from module `diag5385`
-fail_compilation/diag5385.d(33): Error: struct `imports.fail5385.S` member `privX2` is not accessible
-fail_compilation/diag5385.d(34): Deprecation: `imports.fail5385.S.packX2` is not visible from module `diag5385`
-fail_compilation/diag5385.d(34): Error: struct `imports.fail5385.S` member `packX2` is not accessible
+fail_compilation/diag5385.d(19): Error: no property `privX` for type `imports.fail5385.C`, did you mean `imports.fail5385.C.privX`?
+fail_compilation/diag5385.d(20): Error: no property `packX` for type `imports.fail5385.C`, did you mean `imports.fail5385.C.packX`?
+fail_compilation/diag5385.d(21): Error: no property `privX2` for type `imports.fail5385.C`, did you mean `imports.fail5385.C.privX2`?
+fail_compilation/diag5385.d(22): Error: no property `packX2` for type `imports.fail5385.C`, did you mean `imports.fail5385.C.packX2`?
+fail_compilation/diag5385.d(23): Error: no property `privX` for type `S`, did you mean `imports.fail5385.S.privX`?
+fail_compilation/diag5385.d(24): Error: no property `packX` for type `S`, did you mean `imports.fail5385.S.packX`?
+fail_compilation/diag5385.d(25): Error: no property `privX2` for type `S`, did you mean `imports.fail5385.S.privX2`?
+fail_compilation/diag5385.d(26): Error: no property `packX2` for type `S`, did you mean `imports.fail5385.S.packX2`?
```

Comment:
Okay I didn't have my morning coffee yet, but those suggestions don't make much sense to me. Either we should say it's not visible, or just not suggest the invisible symbol in the suggestion.

Reply:
I'm planning to remove the suggestion.

Reply:
Removed.